frontend/src/pages/mocap/MocapSubjectView.tsx

```diff
@@ -998,6 +1000,22 @@ const MocapSubjectView = observer((props: MocapSubjectViewProps) => {
             props.cursor.subjectJson.setAttribute("residualsToZero", e.target.checked);
           }}></input>
         </div>
+        <div className="mb-15">
+          <p>
+            Run a Moco Inverse problem to get muscle forces, after the dynamics optimization is complete:{" "}
+          </p>
+          <input type="checkbox" checked={runMoco} onChange={(e) => {
```

Comment:
Is it possible to have the "exportMoco" box automatically check if "runMoco" is checked? Because that is the backend behavior.

Reply:
Yes, we can do that